src/views/Profile/AdminTable/Modals/SetPublicationDateModal.styles.tsx

```diff
@@ -0,0 +1,36 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+export const useStyles = makeStyles((theme: Theme) => ({
+  wrapper: {
+    marginBottom: theme.spacing(6),
+    position: 'relative',
+  },
+
+  dateRangeIcon: {
+    position: 'absolute',
+    right: '14px',
+    bottom: '16px',
+  },
+
+  datePicker: {
+    position: 'relative',
+
+    '& > label': {
+      backgroundColor: theme.palette.common.white,
+      margin: '8px 0 0 18.5px',
+      padding: '0 3px',
+      zIndex: '10',
+    },
+  },
+
+  currentDate: {
+    display: 'inline-block',
+    border: 'solid 1px #c4c4c4',
+    borderRadius: '4px',
+    padding: '18.5px 14px',
```

Comment:
fractional part in `padding` looks suspicious. Why is it needed here?

Reply:
I made it the same as it is on other modal forms, but I can take 0.5px away, hope QA won't notice it ))